consensus.go

```diff
@@ -159,10 +173,15 @@ type Pbft struct {
 	// tracer is a reference to the OpenTelemetry tracer
 	tracer trace.Tracer
 
-	// calculates timeout for a specific round
+	// roundTimeout calculates timeout for a specific round
 	roundTimeout RoundTimeout
 
-	forceTimeoutCh bool
+	// notifier is a reference to the struct which encapsulates handling messages and timeouts
+	notifier StateNotifier
+}
+
+func (p *Pbft) Log(log string) {
```

Comment:
If it turns out that logging within the function is not needed, then this function can be removed as well (we can add it later on, once we see that we really need it).
`func (n *node) isStuck(num uint64) (uint64, bool)`

Reply:
Yep. Its a leftover from debugging. Will remove it.